.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ steps: &steps
   steps:
     - checkout
     - run: bundle install
-    - run: ruby .travis.rb
+    - run: ./.travis.sh
```

Comment:
Or `bash travis.sh`. Probaby we should rename this to `ci.sh` or something like this.

Reply:
I actually advised @deivid-rodriguez against renaming the file just yet. There are still a few Travis specific things left in that file.

Reply:
Absolutely, I have that change in the [original branch](https://github.com/rubocop-hq/rubocop/commit/4a5a073c51998668108f760303e47565603dd9fa), but I wanted to compile a minimal diff here.

Regarding `bash travis.sh`, :+1:. I'll leave the shebang as well in case someone wants to run it using the shortcut.